src/views/EditNote/EditNote.tsx

```diff
@@ -95,6 +99,7 @@ export class EditNote extends React.Component<Props> {
                         Save changes
                     </button>
                 </nav>
+                <div>{this.props.failNote}</div>
```

Comment:
You can use destructuring to avoid using this.props explicitly.
You can do const { failNote } = this.props;
And then just use it as {failNote}